client/src/components/DataFiles/DataFilesModals/DataFilesCopyModal.jsx

```diff
@@ -58,7 +58,11 @@ const DataFilesCopyModal = React.memo(() => {
       (s) => s.hidden || (s.scheme !== 'private' && s.scheme !== 'projects')
     )
     .filter((s) => !(s.scheme === 'public' && canMakePublic))
-    .map((s) => s.system);
+    .map((s) => `${s.system}${s.homeDir || ''}`);
```

Comment:
It seems to work as expected but do these changes need to make it to DataFilesSelectModal too?  if so, in this PR or follow on Jira issue?

Reply:
Yeah they should be in there as well. I think it can be part of this PR since this PR in general is addressing the same issue of the correct systems not appearing where they should be. I'll update this shortly